src/elements/content-sidebar/Sidebar.js

```diff
@@ -76,8 +83,32 @@ class Sidebar extends React.Component<Props, State> {
     constructor(props: Props) {
         super(props);
 
+        const hasActivity = SidebarUtils.canHaveActivitySidebar(props);
```

Comment:
We have very similar code to register routes for each sidebar in `SidebarPanels`. I think we'll need to rethink our approach to keep the logic for routing in one place.